ui/apps/platform/src/utils/searchUtils.ts

```diff
@@ -194,28 +194,20 @@ export function flattenFilterValue<UndefinedFallback>(
  *
  * @param searchFilter The `SearchFilter` to apply to the list query
  * @param sortOption The field to sort results by and whether to sort ascending or descending
- * @param page The page offset to return
- * @param pageSize The number of items per page
+ * @param page The page offset to return, pages are 1-indexed
+ * @param perPage The number of items per page
  */
 export function getListQueryParams(
     searchFilter: SearchFilter,
     sortOption: ApiSortOption,
-    page?: number,
-    pageSize?: number
+    page: number,
+    perPage: number
```

Comment:
With 9 results in 4 files for function call (and most of them in unit tests) what are pro and con to follow example of `getPaginationParams` to replace arguments by position with argument as configuration object with (now consistent and) meaningful property names.

Reply:
Does only calls in AlertsService.ts and VulnerabilityExceptionService.ts file suggest maybe other services or containers are duplicating some of this logic?

Reply:
Yes I think moving to config object is a great idea, and should be a small change I can add onto this PR.

I do suspect there is more logic duplication around, and I was able to get another chunk in https://github.com/stackrox/stackrox/pull/12750. There are some places where logic is slightly altered due to edge cases or unusual requirements (e.g. Policies client side pagination) as well. I haven't done an exhaustive check at this point yet though.